change/@fluentui-eslint-plugin-52e3c027-6b48-4277-8c18-c42a5bd64756.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Enable exhaustive-deps rule for useIsomorphicLayoutEffect",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "behowell@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
Largely a note to self, but also if you happen to do a lint plugin change again...I think to avoid the deluge of unnecessary version bumps (since this is only ever a dev dep), it would be a good idea to manually set `dependentChangeType` to `none`. 🤦‍♀️ (Not saying you did anything wrong, you just happened upon a case I hadn't thought about in awhile which makes some tooling limitations obvious.)